code/datums/craft/recipes/appliances.dm

```diff
@@ -63,6 +63,17 @@
 	flags = CRAFT_ON_FLOOR|CRAFT_ONE_PER_TURF
 	related_stats = list(STAT_MEC)
 
+/datum/craft_recipe/appliance/tint_button
+	name = "window tint controller"
+	result = /obj/machinery/button/windowtint
+	time = WORKTIME_NORMAL
+	steps = list(
+		list(/obj/item/device/assembly/signaler)
```

Comment:
```suggestion
		list(/obj/item/device/assembly/signaler, 1)
```
This is tested right?

Reply:
I did indeed test it I believe, but I can give it another quick look when I get home.